runtime/src/lib.rs

```diff
@@ -177,6 +178,14 @@ fn enter_idle(drivers: &mut Drivers) {
     caliptra_cpu::csr::mpmc_halt_and_enable_interrupts();
 }
 
+fn human_readable_command(bytes: &[u8]) -> Option<&str> {
+    if bytes.len() != 4 || bytes.iter().any(|c| !c.is_ascii_alphanumeric()) {
+        None
+    } else {
+        core::str::from_utf8(bytes).ok()
```

Comment:
This will add about ~900 bytes to the code size, we still have a lot of budget left but we may want to consider removing this if we start running into the limit.